style.css

```diff
@@ -910,4 +1003,97 @@ header {
     height: 550px;
     background: #3c3a39;
   }
+
+  .show-inactive {
+    position: fixed;
+    z-index: 100;
+    display: none;
+    top: 7%;
+    left: 7%;
+    right: 7%;
+    background: #fff;
+    bottom: 7%;
+    transform: none;
+  }
```

Comment:
Great job @Solobarine 🤝

- [ ] [Optional] You may kindly blur the backgrounds of both the mobile popup and the desktop popup so that your design can be close to the replica of the Figma template.
- [ ] [Optional] You may also add a margin-bottom to both your desktop and mobile popup
- [ ] [Optional] You may kindly increase the size of the cancel button for both the mobile and the desktop popup so that your design is a clear replica of the Figma design.
- Your design
![mobile popup](https://user-images.githubusercontent.com/63917725/181736263-be5e5593-8cd0-459f-992d-7020f171cfb4.PNG)

- Figma design
![Figma design](https://user-images.githubusercontent.com/63917725/181736442-1791aa19-eca1-4583-a91f-ead67d1785ab.PNG)

Reply:
Thanks Gordon, I will implement them